tests/phpunit/src/custom/gradebook_fake/RecordManagerTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+use Sugarcrm\Sugarcrm\custom\gradebook_fake\RecordManager;
+use PHPUnit\Framework\TestCase;
+
+class RecordManagerTest extends TestCase
```

Comment:
PHPdoc blocks.

Reply:
Are you suggesting adding PHPdoc for the tests?